VaccinationSystem/VaccinationSystem/Data/VaccinationStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace VaccinationSystem.Data
+{
+    public enum VaccinationStatus
```

Comment:
To można przenieść do Vaccine i tam dodać pole typu enum. Baza danych powinna to już ogarnąć.